### PR TITLE
docs(cli,csv,datetime,fmt,fs,http,ini,semver,testing): assert optional properties on types/interfaces have `@default` tag

### PR DESCRIPTION
**Pull request by iuioiua**
This change adds support for asserting that all optional interface properties have a `@default` JSDoc tag defined to the doc linter.

It seems that the most appropriate tag for some of these properties is `@default {undefined}`. However, I consider this boilerplate. So, I'm thinking of removing these lines across the codebase, then disabling this new assertion. We can then re-enable the assertion whenever we'd like to update these `@default` JSDoc tags again. WDYT? Happy to hear any ideas on how we can ignore this rule.

### CODE REVIEW

**Comment by iuioiua** [comment]
PTAL @kt3k

File: fs/expand_glob.ts [hunk 1]
```diff
@@ -19,9 +19,17 @@ const isWindows = Deno.build.os === "windows";
 
 /** Options for {@linkcode expandGlob} and {@linkcode expandGlobSync}. */
 export interface ExpandGlobOptions extends Omit<GlobOptions, "os"> {
-  /** File path where to expand from. */
+  /**
+   * File path where to expand from.
+   *
+   * @default {Deno.cwd}
```
Comment (kt3k) [inline fs/expand_glob.ts:25]:
```suggestion
   * @default {Deno.cwd()}
```

**Review by kt3k — APPROVED** [review]
LGTM

File: async/deadline.ts [hunk 2]
```diff
@@ -8,7 +8,11 @@ import { abortable } from "./abortable.ts";
 
 /** Options for {@linkcode deadline}. */
 export interface DeadlineOptions {
-  /** Signal used to abort the deadline. */
+  /**
+   * Signal used to abort the deadline.
+   *
+   * @default {undefined}
```
Comment (iuioiua) [inline async/deadline.ts:14]: `undefined` will now show up in docs as the default value in quite a few of these cases. I consider that boilerplate and wanted to remove those instances before merging this. Does that all sound good, @kt3k?

Comment (kt3k) [reply async/deadline.ts:14]: Sounds reasonable, but doesn't that cause a problem with the linter?

Comment (kt3k) [reply async/deadline.ts:14]: Sounds good

> I'm not yet sure how else we can direct the linter to ignore a property.

Let's revisit that later

**Pull request merged**